packs/unwanted-chrome-extensions.conf

```diff
@@ -0,0 +1,65 @@
+{
+  "platform": "windows,darwin",
+  "queries": {
+    "BetternetVPN": {
+      "query": "SELECT * FROM users JOIN chrome_extensions USING (uid) WHERE identifier='gjknjjomckknofjidppipffbpoekiipm';",
+      "interval": 86400,
```

Comment:
Just my 2-cents, but what about doing `3600` for the interval? I feel like for chrome extensions they could be installed pretty quick, and considering this query will likely be enumerating information on client user laptops this query will likely only run once every 3 business days. Thoughts?

Reply:
Agreed, just had 86400 to be safe :)